bleak/backends/bluezdbus/scanner.py

```diff
@@ -304,4 +303,38 @@ def parse_msg(self, message):
         )
 
         if self._callback is not None:
-            self._callback(message)
+            # First ensure that the message body is a list and has the dict where we can find the information we need
+            if isinstance(message.body, list) and len(message.body) > 2:
+                discovery_data = message.body[1]
+
+                # Get all the information wanted to pack in the advertisement data
+                _address = message.path[message.path.rfind("dev_") + 4 :].replace(
+                    "_", ":"
+                )
+
+                _local_name = discovery_data.get("Name", None)
```

Comment:
This could be 

    _local_name = discovery_data.get("Name", "Unknown")

instead of the special case for  `_local_name`  below